src/launchpad/kafka.py

```diff
@@ -113,16 +176,12 @@ def create_with_partitions(
         next_step: ProcessingStrategy[Any] = CommitOffsets(commit)
         logger.info("KAFKASETUP: Base strategy: CommitOffsets")
 
-        # Add healthcheck if configured
         logger.info("KAFKASETUP: Checking healthcheck configuration...")
-        if self.healthcheck_file:
-            logger.info(f"KAFKASETUP: Healthcheck file configured: {self.healthcheck_file}")
-            logger.info("KAFKASETUP: Adding Healthcheck strategy to processing chain")
-            next_step = Healthcheck(self.healthcheck_file, next_step)
-            logger.info("KAFKASETUP: Healthcheck strategy added successfully")
-        else:
-            logger.warning("KAFKASETUP: No healthcheck file configured - skipping healthcheck strategy")
-            logger.info("KAFKASETUP: Processing will continue without healthcheck monitoring")
+        logger.info(f"KAFKASETUP: Healthcheck file configured: {self.healthcheck_file}")
+        logger.info("KAFKASETUP: Adding Healthcheck strategy to processing chain")
+        assert self.healthcheck_file
+        next_step = Healthcheck(self.healthcheck_file, next_step)
+        logger.info("KAFKASETUP: Healthcheck strategy added successfully")
```

Comment:
im not wed to keeping these now that kafka appears to be stable but it doesn't fire too often so not a big deal